src/pybind/mgr/dashboard/tox.ini

```diff
@@ -6,7 +6,8 @@ envlist =
     check,
     run,
     apidocs,
-    openapi-{check,fix, doc}
+    openapi-{check,fix, doc},
+    grafonnet-check
```

Comment:
You should provide both the check and fix actions, same as with openapi. The fix is the generate one (it simply overwrites the existing json).

Reply:
Sure, will do that.

Reply:
You need to expose the -fix here too, or tox won't allow you to call it.

Reply:
Sure